Assets/BossRoom/Scripts/Client/com.unity.multiplayer.samples.bossroom.client.asmdef

```diff
@@ -11,7 +11,9 @@
         "Unity.Netcode.Adapter.UTP",
         "Photon Realtime Transport for Netcode for GameObjects",
         "Unity.Services.Relay",
-        "Unity.Multiplayer.Samples.Utilities"
+        "Unity.Multiplayer.Samples.Utilities",
+        "Unity.Services.Lobbies",
+        "ParrelSync"
```

Comment:
mmm just to be sure, does that compile with release builds? I thought PS's assemblies were editor only?